handler_test.go

```diff
@@ -337,7 +337,9 @@ func TestHandler_Users_OneUser(t *testing.T) {
 	s := NewHTTPServer(srvr)
 	defer s.Close()
 
-	status, body := MustHTTP("GET", s.URL+`/users`, nil, nil, "")
+	query := map[string]string{"q": "LIST USERS"}
+	status, body := MustHTTP("GET", s.URL+`/query`, query, nil, "")
+
 	if status != http.StatusOK {
 		t.Fatalf("unexpected status: %d", status)
 	} else if body != `[{"name":"jdoe","admin":true}]` {
```

Comment:
Shouldn't the output format follow the JSON format of the `*influxql.Row` result set?